faq.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Responsive grid project</title>
+        <link rel="stylesheet" href="./css/header.css"/>
+        <link rel="stylesheet" href="./css/faq.css"/>
+        <link rel="stylesheet" href="./css/footer.css"/>
```

Comment:
I really like how you break down the CSS files as you are starting to think in terms of components